src/Console/Arguments.php

```diff
@@ -183,8 +224,24 @@ public function getBooleanOption(string $name): ?bool
      * Gets a multiple option's value or null if not set.
      *
      * @return list<string>|null
+     * @deprecated
```

Comment:
Should also list the reason here in docblock for tools to parse it out.
```suggestion
     * @deprecated 5.2.0 ...
```